repository/b.json

```diff
@@ -32,6 +32,10 @@
 				{
 					"sublime_text": ">=3000",
 					"tags": true
+				},
+				{
+					"sublime_text": ">=4000",
+					"tags": "st4-"
```

Comment:
Also, would it make sense to make the ST3 use the tag prefix and make the ST4 use just plain version tags since ST4 release is supposed to be just around the corner?

Reply:
There are already a large number of non-prefixed tags in the repository, and I think it makes the most sense to use a new prefix going forward rather than renaming all of the existing tags.

Reply:
I don't see why the old ones would need to be renamed.

If the master branch is currently compatible with ST3 then fork off it to create an ST3 branch and then use the master to develop the ST4-compatible version.

Then create an ST3-prefixed tag for ST3 and make a switch here to use it.

Reply:
I've changed it.